test/helpers/mock_loader.rb

```diff
@@ -20,7 +20,7 @@ class MockLoader
     macos10_10: { name: "mac_os_x", family: "darwin", release: "10.10.4", arch: nil },
     macos10_16: { name: "darwin", family: "darwin", release: "10.16", arch: nil },
     ubuntu1404: { name: "ubuntu", family: "debian", release: "14.04", arch: "x86_64" },
-    ubuntu: { name: "ubuntu", family: "debian", release: "20.04", arch: "x86_64" },
+    ubuntu: { name: "ubuntu", family: "debian", release: "22.04", arch: "x86_64" },
```

Comment:
We want to keep 20.04 and 22.04 both and remove the Versions which is EOL like Ubuntu 14.04. Could you please confirm this?